fluent-plugin-events/test/resources/api_list_events_v1.json

```diff
@@ -0,0 +1,171 @@
+{
+    "kind": "EventList",
```

Comment:
Is there no event type like `ADDED` or `MODIFIED` specified in the `EventList` resource type?

Reply:
`Event` is a resource type, and `ADDED` or `MODIFIED` types are only for change of resources, eg. change of events, change of endpoints etc. This is also why you don't see `ADDED` or `MODIFIED` in `EndpointList`, but only when you watch for endpoint events (changes)

Reply:
Thanks for clarifying Maisie! I appreciate the help